recipe/build.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 export CC=$(basename "$CC")
+export ASPP="$CC -c"
 export AS=$(basename "$AS")
 export OCAML_PREFIX=$PREFIX
 export OCAMLLIB=$PREFIX/lib/ocaml
 
-bash -x ./configure -prefix $OCAML_PREFIX -cc $CC -aspp "$CC -c" -as "$AS"
+bash -x ./configure -prefix $OCAML_PREFIX
```

Comment:
This makes sense - you're using the CC, AS and new ASPP environment variables instead, right?

Reply:
Yes. These options are not supported anymore apparently, but `./configure --help` says that the environment variables that you mentioned are taken into account.

Reply:
So that change is prompted by updating from 4.06.1 to 4.10.2 - understood, thanks.